e2e/spice/spicedb.go

```diff
@@ -76,6 +76,7 @@ func (s *Node) Start(ctx context.Context, logprefix string, args ...string) erro
 		"./spicedb",
 		"serve",
 		"--log-level=debug",
+		"--datastore-request-hedging=false",
```

Comment:
does this need an option func?

Reply:
I've only been adding option funcs for things that need to vary between test runs; so I would add one if we start using this to run other e2e tests where we want request hedging

Reply:
Added one for this